src/test/java/com/wikia/webdriver/testcases/chattests/ChatTests.java

```diff
@@ -255,41 +261,25 @@ public void userCanBeKickedOutFromChat() {
     Assertion.assertTrue(chatUserOne.isUserKickedFromChat(), "USER IS NOT KICKED FROM CHAT");
   }
 
-  @Test
+  @Test(groups = {"ChatTestsForUser_012"})
   public void bannedUserCanNotEnterTheChat() {
+
     ChatPage chatUserStaff = openChatForUser(userStaff, userStaffPassword);
 
     switchToWindow(1);
     SpecialVersionPage chatWindow = new SpecialVersionPage().open();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "chatWindow" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1481)